docs/index.md

```diff
@@ -1063,6 +1064,12 @@ tcp://192.168.99.109:2376
 #### Amazon Web Services
 Create machines on [Amazon Web Services](http://aws.amazon.com).  You will need an Access Key ID, Secret Access Key and a VPC ID.  To find the VPC ID, login to the AWS console and go to Services -> VPC -> Your VPCs.  Select the one where you would like to launch the instance.
 
+Use `amazonec2` for the `--driver` and include the 3 required parameters. Depending upon the configured VPC, you may have to specify `--amazonec2-zone` if a subnet does not exist in Availability Zone `a`.
```

Comment:
Thanks @kacole2 then, how about:

--> https://gist.github.com/moxiegirl/a7fe776b7c4127f97fd7

#### Amazon Web Services

To create machines on [Amazon Web Services](http://aws.amazon.com) (AWS), you must supply three required parameters:

* Access Key ID
* Secret Access Key
* VPC ID

You obtain these values from AWS.  To find the VPC ID:

1. Login to the AWS console 
2. Go to **Services -> VPC -> Your VPCs**.  
3. Locate the VPC ID you want from the **VPC** column. 
4. Examine the *Availability Zone** column to verify that zone `a` exists and matches the VPC ID.
  
    For example, `us-east1-a` is in the `a` availability zone. If the `a` zone is not present, you can create a new subnet in that zone or specify a different zone when you create the machine.

To create the machine instance, specify `--driver amazonec2` and the three required parameters. 

```
$ docker-machine create --driver amazonec2 --amazonec2-access-key AKI******* --amazonec2-secret-key 8T93C********* --amazonec2-vpc-id vpc-****** aws01
```

This example assumes the ID was found in the `a` availability zone. Use the `--amazonec2-zone` flag to specify a zone other than the `a` zone. For example, `--amazonec2-zone c` specifies the `c` zone.